statistics/handle/update.go

```diff
@@ -998,12 +998,12 @@ var AutoAnalyzeMinCnt int64 = 1000
 // TableAnalyzed checks if the table is analyzed.
 func TableAnalyzed(tbl *statistics.Table) bool {
 	for _, col := range tbl.Columns {
-		if col.Count > 0 {
+		if col.StatsVer != statistics.Version0 {
```

Comment:
Why not use `col.StatsCollected`?

Reply:
If the column is added and its stats are generated according to its default value, `StatsCollected` returns true but the column is not analyzed. I will add another function called `IsAnalyzed` for better encapsulation.